docusaurus/docs/Android/04-compose/02-component-architecture.mdx

```diff
@@ -106,7 +109,8 @@ They offer fully customizable behavior as well as customizable UI where it makes
 
 * `Avatar`: Shows an image provided as a parameter, in a shape defined by the `ChatTheme`. Fully customizable in terms of size, shape, alignment and more.
 * `SearchInput`: Shows a leading icon for search, an input field to write your query and a trailing icon to clear the input. The leading icon can be fully customized, as well as the listeners for state changes.
-* `MessageList` and `ChannelList`: We offer stateless alternatives to some `ViewModel` powered components, if you don't want to use our `ViewModel` and want to customize the behavior, as well as the UI. 
+* `MessageList` and `ChannelList`: We offer stateless alternatives to some `ViewModel` powered components, if you don't want to use our `ViewModel` and want to customize the behavior, as well as the UI.
+* `MessageComposer`: A stateless alternative not backed by a `ViewModel`. Useful if you want more control over the component.
```

Comment:
While I didn't want to go too far with listing stateless components, I felt that MessageComposer was important enough to warrant a place on this list seeing how its one of the central components in our SDK

Reply:
Yeah makes sense, good call!